launch/publish_demo_data.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/python3
```

Comment:
This file doesn't seem to be installed

Reply:
I had this file installed and moved to using `env python3`. Since humble is python3 I hope that isn't an issue.